client/src/libs/jquery/jquery.wymeditor.js

```diff
@@ -28,24 +28,37 @@
    Namespace: WYMeditor
    Global WYMeditor namespace.
 */
-if(!WYMeditor) var WYMeditor = {};
+if (!WYMeditor) var WYMeditor = {};
```

Comment:
@mvdbeek Something went wrong with your prettier config here, this shouldn't have been touched?